miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -743,11 +702,7 @@ public static SampleValidRelationshipDto asDto(SampleValidRelationship from) {
   }
 
   public static Set<SampleValidRelationshipDto> asSampleValidRelationshipDtos(Set<SampleValidRelationship> from) {
-    Set<SampleValidRelationshipDto> dtoSet = Sets.newHashSet();
-    for (SampleValidRelationship sampleValidRelationship : from) {
-      dtoSet.add(asDto(sampleValidRelationship));
-    }
-    return dtoSet;
+    return from.stream().map(Dtos::asDto).collect(Collectors.toSet());
   }
 
   public static SampleValidRelationship to(SampleValidRelationshipDto from) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "from". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)